app/livechat/imports/server/rest/dashboards.ts

```diff
@@ -13,29 +11,25 @@ import {
 	getAgentsProductivityMetrics,
 	getChatsMetrics,
 } from '../../../server/lib/analytics/dashboards';
-import { Users } from '../../../../models';
+import { Users } from '../../../../models/server';
 
 API.v1.addRoute(
 	'livechat/analytics/dashboards/conversation-totalizers',
 	{ authRequired: true },
 	{
-		get() {
+		async get() {
 			if (!hasPermission(this.userId, 'view-livechat-manager')) {
 				return API.v1.unauthorized();
 			}
 			let { start, end } = this.requestParams();
 			const { departmentId } = this.requestParams();
 
-			check(start, String);
-			check(end, String);
-			check(departmentId, Match.Maybe(String));
-
-			if (isNaN(Date.parse(start))) {
+			if (!start) {
```

Comment:
Any reason to remove the "checks"? We use them to validate the actual type of the param. Params like "dates" are strings, we convert them to be actual dates on the server.

Reply:
I thought they were unnecessary once I implemented TS interfaces 🤦

Reply:
I just brought them back in 5b3bc8b74949a9350d52c3d9f288bd809bbcdf02